flycheck.el

```diff
@@ -10,7 +10,7 @@
 ;; URL: http://www.flycheck.org
 ;; Keywords: convenience, languages, tools
 ;; Version: 33-cvs
-;; Package-Requires: ((dash "2.12.1") (pkg-info "0.4") (let-alist "1.0.4") (seq "1.11") (emacs "24.3"))
+;; Package-Requires: ((emacs "26.1") (dash "2.12.1") (pkg-info "0.4") (let-alist "1.0.4") (seq "1.11"))
```

Comment:
The default version on Ubuntu 18.04 is still `25.x`, but it will only support until April 2023. I hope this is reasonable changes. LMKWYT?

Related to #1970.

Reply:
Yeah, that's fine IMO.

Reply:
No strong feelings, but would there be a downside to using 25.3?  I have a few old machines on which I use 25.

Reply:
I guess as a middle ground we can target 25.x, but omit it from the CI. At any rate - I don't feel strongly about this.

Reply:
> No strong feelings, but would there be a downside to using 25.3? I have a few old machines on which I use 25.

There won't be a downside using `25.x` (user-wise), although Eask only supports `26.1` or above.

I would encourage people to upgrade to `26.1` or above. Emacs has significantly improved throughout the years. Like: 

1. Some useful macros `when-let*` and `if-let*`
2. `let-alist` and `seq ` are built-in now
3. `set` and `make-local-variable` can replace with `defvar-local` or `setq-local`.
4. It's faster

, etc

In terms of maintenance, it's better and easier for us. Overall, it's just a nice Emacs version to have/target.

@cpitclaudel LMKWYT? If you still prefer `25.x`, I will perform the change. ;)

Reply:
Flycheck is too large (I think) to play fast an loose with its supported version set; we have lots of folks that depend on us, and we have traditionally supported a lot of relatively old versions.  I think sticking with 25.x for now would be safer.

(I agree with you that 26 is nicer)

Reply:
👍 I have reverted and now supports `25.1`!